### ISSUE TITLE
customize calendar height

### ISSUE THREAD

**Issue by AsasinCree** [airbnb/react-dates#1641]
**react-dates version**
react-dates@19.0.3

**Describe the bug**
After customize some UI style, **last week** is _covered_ by calendar container like below.
![image](https://user-images.githubusercontent.com/18276301/57603501-7b8da200-7594-11e9-80be-8ce1da9171f9.png)

I tried to override the css class "**DayPicker_transitionContainer**" to fix the height but it won't adjust dynamically according to weeks account(like below two different month, height is different)
![3](https://user-images.githubusercontent.com/18276301/57603757-10909b00-7595-11e9-9c30-31d26b1d5b8b.gif)

I checked the source code and find below logic: 
![image](https://user-images.githubusercontent.com/18276301/57605849-294f7f80-759a-11e9-8900-0fece396a369.png)
I could change the "**MONTH_PADDING**" to adjust the height to fix my issue but "MONTH_PADDING" is a const in this file which can't be controlled from outside.

**What I want is let "MONTH_PADDING" as a prop to us.**

**Comment by iamolegga** [comment]
same problem, @AsasinCree any luck with this?

**Comment by iamolegga** [comment]
my solution:

```tsx

useEffect(() => {
  changeMonthHandler(from || moment());
});

<DayPickerRangeController
  startDate={from}
  endDate={to}
  onNextMonthClick={changeMonthHandler}
  onPrevMonthClick={changeMonthHandler}
/>

let transitionContainerStylesheet = document.createElement('style');
document.head.appendChild(transitionContainerStylesheet);

function changeMonthHandler(date: moment.Moment) {
  transitionContainerStylesheet.innerHTML = `
    .DayPicker_transitionContainer {
      height: ${cellHeight * calcWeeksInMonth(date) + restCalendarContentHeight }px !important;
    }
  `;
}

function calcWeeksInMonth(date: moment.Moment) {
  const dateFirst = moment(date).date(1);
  const dateLast = moment(date).date(date.daysInMonth());
  const startWeek = dateFirst.week();
  const endWeek = dateLast.week();
  if (endWeek < startWeek) {
    return dateFirst.weeksInYear() - startWeek + 1 + endWeek;
  } else {
    return endWeek - startWeek + 1;
  }
}
```

tldr, create stylesheet with `!important` height for container, that will not be redefined by inline height, calculated by library

I've tried to use inline style on `.DayPicker_transitionContainer` but there was some chattering of height, because first it set height from library and then custom height, so Stylesheet solves this problem

**Comment by yournatalita** [comment]
I played with styles during the project and I know I probably did something wrong, but I came to the same problem. 

I realized somehow on some step of my project height started calculating wrong everytime with the same difference from the right way ("one cell height", for me it's 37px)

So i added it as a padding and overrode box-sizing 

```
.DayPicker_transitionContainer {
    padding-bottom: 37px;
    box-sizing: content-box;
}
```
Since I have complex customized layout seems it is the only workaround for me
